### PR TITLE
Artwork cleanup and additions

### PR DESCRIPTION
**Pull request by pgodd**
Includes new kOS Logo and complete Unity Dev package in editable formats

### CODE REVIEW

**Comment by Dunbaratu** [comment]
This PR contains ZIP files in it.

Is that correct?  I fear they may be zips that contain the same thing as the exploded files in the repo (in which case they don't belong in the repo), but I'm not entirely sure as these files are unknown to me.

**Comment by pgodd** [comment]
Cleaned up zip files left over from old release, and changed filename to remove tilde ~ symbol

File: src/kOS/kOS.csproj [hunk 1]
```diff
@@ -30,7 +30,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\..\Resources\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\..\..\Library\Application Support\Steam\SteamApps\common\Kerbal Space Program\KSP.app\Contents\Data\Managed\Assembly-CSharp.dll</HintPath>
```
Comment (erendrake) [inline src/kOS/kOS.csproj]: you might want to revert this change

Comment (hvacengi) [inline src/kOS/kOS.csproj]: It got reverted in @Dunbaratu's review.
